src/components/RadioGroup/RadioGroup.tsx

```diff
@@ -0,0 +1,69 @@
+/**
+ * Copyright 2024 Mia srl
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { Radio as AntRadio, RadioChangeEvent } from 'antd'
+import { ReactElement, ReactNode, useMemo, useState } from 'react'
+
+import { Radio } from './components/Radio'
+import { RadioGroupProps } from './RadioGroup.props'
+import styles from './RadioGroup.module.css'
+
+const { options: optionsClass } = styles
+
+const defaults = {
+  disabled: false,
+}
+
+export const RadioGroup = ({
+  defaultValue,
+  disabled = defaults.disabled,
+  options,
+  onChange,
+}: RadioGroupProps): ReactElement => {
+  const [value, setValue] = useState<string | number | boolean >(defaultValue)
```

Comment:
Why can't RadioGroup directly use the defaultValue prop?

Reply:
I agree, keeping the defaultValue updated onchange sounds like a bit of overhead, is it actually required by antd to work it like this?

Reply:
I thought in Ant examples always used the value state, but actually in the example with the default value is not used. 
I passed the devfaultValue prop directly to Radio.Group and removed the value state, and everything seems to work fine